crates/chronicle/src/bootstrap/cli.rs

```diff
@@ -1003,6 +1003,12 @@ impl SubCommand for CliModel {
                             .takes_value(true)
                             .default_value("127.0.0.1:9982")
                             .help("The graphql server address (default 127.0.0.1:9982)"),
+                    ).arg(
+                        Arg::new("open")
```

Comment:
Does it make enough sense that an "unlock CORS" option is called `--open`?

Reply:
Great point, but for now it would mean making changes to other repositories.

Reply:
"for now" - probably also subsequently, or is there a coming change that'll make it easier? My concern is that it will become further entrenched by more people/scripts using it, thus increasingly much pain to change, so now may be our best opportunity, or we decide it won't cause enough confusion to be worth it. If we're not sure, or for some reason it'll be easier later, fine with me if we just ask in scrum, open a ticket raising the question, etc. rather than impeding this PR.

It's also worth keeping an eye out for possibilities to improve development workflows to make multi-repository changes less inhibiting.

Reply:
All great points, this is just more about unbreaking things. And even better if it raises questions about what we want from this after we get things working again.

Reply:
We could alias the command to make migration easier

Reply:
That way we can take out --open once deps are updated, similarly to how we handle deprecation in library apis

Reply:
I'll move them to a ticket. Just trying to prevent us from unbreaking things in ways we come to regret!

Reply:
Thanks @ryanbtp , I hadn't looked at aliases in clap before. `--unlock-cors` sound good to you as an alias, @mtbc ?

Reply:
Sure, my point is more that we should think about this stuff sooner rather than later: ideally we can name/alias it now something that we're happy with long-term but it's fine to get this PR in first. Feel free to defer all this to when we pick up CHRON-254.

As it *does* open something, it might also help simply to explain a bit more in the longer help text. Personally, I would have been surprised to find what's simply an "Unlock CORS" option opening a "method not allowed" browser tab at me but it's been a while since I had to focus on web security.

I guess it depends what it is *doing*. How does opening a browser tab unlock CORS? Perhaps there's a succinct statement there that can inspire the name or description. I'm fine for that to go into a later PR (or for us to decide that `--open` is fine) just so long as we get around to it before everyone's gotten used to the current.

Reply:
It turned out that not least of the confusion here was being caused by having not put back all the parts necessary to do what the PR says, but that's all there now and we're back with the original functionality of being able to use the graphql playground.

Reply:
@mtbc I've also documented what this is useful for now.